plugins/jobs/src/js/components/JobsBreadcrumbs.js

```diff
@@ -69,56 +49,67 @@ function getItemSchedule(item) {
   );
 }
 
-function getBreadcrumb(item, details = true) {
-  const id = item.getId();
-  const name = item.getName();
-  const link =
-    item instanceof Job ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
+function getBreadcrumb(item, id = "", name = "") {
+  console.log("getBreadcrumb", id);
+  const isDetailPage = item.id === id;
+  const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
 
   return (
     <Breadcrumb key={id} title="Jobs">
       <BreadcrumbTextContent>
         <Link to={link}>{name === "" ? "Jobs" : name}</Link>
       </BreadcrumbTextContent>
-      {details ? getItemSchedule(item) : null}
-      {details ? getItemStatus(item) : null}
+      {/* {details ? getItemSchedule(item) : null */}
```

Comment:
<ItemStatus /> just show status. On the past we had schedule and status shown here, but it seems commented out. Is this correct? I think if it is we should remove line 62